src/applications/edu-benefits/complaint-tool/helpers.js

```diff
@@ -0,0 +1,30 @@
+import environment from '../../../platform/utilities/environment';
+
+export function transformAutocompletePayloadForAutosuggestField({ payload, error, searchTerm }) {
+  if (payload) {
+    return {
+      options: payload
+      .data
+      .map(({ value, label }) => ({ value, label })),
+      searchTerm
+    };
+  }
+  return [];
+}
+export function fetchAutocompleteSuggestions(text, version) {
```

Comment:
Pulling from the existing end point- this will be updated once the new autosuggest / search are available.